### DISCUSSION TITLE
Allow reactive b-table sorting to be blocked

### DISCUSSION THREAD

**Discussion by williamjulianvicary** [bootstrap-vue/bootstrap-vue#5848] [Q&A] [answered]
I have a use case where I'm making frequent adjustments to the underlying data in a table via sockets. This is all working well, however I'd like to only re-sort/order the table upon user interaction (in order to stop data bouncing around causing a poor UX).

To be clear, the data can be updated - but if say a field is being edited by a user is updated to "100" up from "1" and this means the order of the data is different, it'll cause the field to move for the user which doesn't give a great user experience.

I had a good look and couldn't see anything available at present that supported this? Unless I'm mistaken?

**Comment by jacobmllr95** [comment, marked as answer]
I think you are better off then to handle the sort yourself and disable local sorting.
See the [Disable local sorting](https://bootstrap-vue.org/docs/components/table#disable-local-sorting) section fo the `<b-table>` docs.
